core/src/test/scala/io/projectglow/GlowSuite.scala

```diff
@@ -72,13 +72,35 @@ class GlowSuite extends GlowBaseTest {
         ("snake_animal", "snake"))
     checkTransform(df)
   }
+
+  test("accept non-string values") {
```

Comment:
I think we could make this test more effective by having the transformer output something that demonstrates the number was parsed properly by the mapper. For example, by checking that the same input parameter passed as a string or as a float results in the same output DF.

Reply:
Isn't that what the following test does? It verifies that an argument passed as a float can be correctly interpreted. I added a line that checks that a string works the same way.